### PR TITLE
MINOR: fix cleanup phase for KStreamWindowAggregateTest

### PR DESCRIPTION
**Pull request by mjsax**
fixes:
```
java.nio.file.NoSuchFileException: /tmp/test7863510415433793941/topic2-Canonized/topic2-Canonized-197001010000/000015.sst
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileAttributeViews$Basic.readAttributes(UnixFileAttributeViews.java:55)
	at sun.nio.fs.UnixFileSystemProvider.readAttributes(UnixFileSystemProvider.java:144)
	at sun.nio.fs.LinuxFileSystemProvider.readAttributes(LinuxFileSystemProvider.java:97)
	at java.nio.file.Files.readAttributes(Files.java:1686)
	at java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:105)
	at java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:199)
	at java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:199)
	at java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:199)
	at java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:69)
	at java.nio.file.Files.walkFileTree(Files.java:2602)
	at java.nio.file.Files.walkFileTree(Files.java:2635)
	at org.apache.kafka.common.utils.Utils.delete(Utils.java:555)
	at org.apache.kafka.streams.kstream.internals.KStreamWindowAggregateTest.testJoin(KStreamWindowAggregateTest.java:320)
```

### CODE REVIEW

**Comment by mjsax** [comment]
Call for review @dguy @enothereska

**Review by dguy — APPROVED** [review]
LGTM

**Comment by mjsax** [comment]
Fix for failing test is #2781

File: streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregateTest.java [hunk 1]
```diff
@@ -49,112 +48,106 @@
...
-                               MockAggregator.TOSTRING_ADDER,
-                               TimeWindows.of(10).advanceBy(5),
-                               strSerde, "topic1-Canonized");
-
-            MockProcessorSupplier<Windowed<String>, String> proc2 = new MockProcessorSupplier<>();
-            table2.toStream().process(proc2);
-
-            driver = new KStreamTestDriver(builder, baseDir);
-
-            setRecordContext(0, topic1);
```
Comment (enothereska) [inline streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregateTest.java:87]: Why are we deleting all these lines? And then reinserting? Is there a formatting change?

Comment (dguy) [reply streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregateTest.java:87]: the `try{...} finally` has been removed

**Review by dguy — APPROVED** [review]
Thanks @mjsax, LGTM

**Comment by junrao** [comment]
@mjsax : Thanks for the patch. LGTM

**Pull request closed without merging**